python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -480,8 +477,10 @@ class RandomForestClassifier(Base):
             input_to_dev_array(X, order='F')
 
         if self.dtype == np.float64:
-            warnings.warn("In order to run predict on the GPU convert"
-                          " the data to float32")
+            warnings.warn("Model is being trained with float64 data, which \
```

Comment:
Does this actually force to CPU mode? I don't see that.

Reply:
It will in the `predict` method in line 688